src/Symfony/Component/Routing/Loader/PhpFileLoader.php

```diff
@@ -48,7 +50,7 @@ public function load($file, string $type = null)
 
         if (\is_object($result) && \is_callable($result)) {
             $collection = new RouteCollection();
-            $result(new RoutingConfigurator($collection, $this, $path, $file));
+            $result($this->getConfigurator($collection, $path, $file));
```

Comment:
I'd suggest making the new method responsible for calling $result:
let's do `$this->callConfigurator($result, $collection, $path, $file)` instead?

Reply:
I've done `$collection = $this->callConfigurator($result, $path, $file);` instead.